doc/releases/reef.rst

```diff
@@ -0,0 +1,219 @@
+  ``debug_rgw``, but it can be controlled independently.
+* RGW: The pubsub functionality for storing bucket notifications inside Ceph is
+  removed. Neither it nor the pubsub zone should be used anymore. The following
+  have also been removed: REST operations, radosgw-admin commands for
+  manipulating subscriptions, and commands for fetching and acking the
+  notifications. 
+  
+  If the endpoint to which the notifications are sent is down or disconnected,
+  we recommend that you use persistent notifications to guarantee their
+  delivery. If the system that consumes the notifications has to pull them
+  (instead of the notifications being pushed to the system), use an external
+  message bus (for example, RabbitMQ or Kafka) for that purpose. 
+* RGW: The serialized format of notification and topics has changed. This means
+  that new and updated topics will be unreadable by old RGWs. We recommend
+  completing the RGW upgrades before creating or modifying any notification
+  topics.
+* RGW: Compression is now supported for objects uploaded with Server-Side
+  Encryption. When both compression and encryption are enabled, compression is
+  applied before encryption. Earlier releases of multisite do not replicate
+  such objects correctly, so all zones must upgrade to Reef before enabling the
+  `compress-encrypted` zonegroup feature: see
+  https://docs.ceph.com/en/reef/radosgw/multisite/#zone-features and note the
+  security considerations.
+  
+Unsorted
+~~~~~~~~
+
+* The ``AT_NO_ATTR_SYNC`` macro is deprecated. Use the standard 
+  ``AT_STATX_DONT_SYNC`` macro. The ``AT_NO_ATTR_SYNC`` macro will be removed
+  in the future.
+* The ``perf dump`` and ``perf schema`` commands have been deprecated in 
+  favor of the new ``counter dump`` and ``counter schema`` commands. These new
+  commands add support for labeled perf counters and also emit existing
+  unlabeled perf counters. Some unlabeled perf counters became labeled in this
+  release, and more will be labeled in future releases; such converted perf
+  counters are no longer emitted by the ``perf dump`` and ``perf schema``
+  commands.
+* Telemetry: Users who have opted in to telemetry can also opt in to
+  participate in a leaderboard in the telemetry public dashboards
+  (https://telemetry-public.ceph.com/). In addition, users are now able to
+  provide a description of their cluster that will appear publicly in the
+  leaderboard. For more details, see:
+  https://docs.ceph.com/en/latest/mgr/telemetry/#leaderboard. To see a sample
+  report, run ``ceph telemetry preview``. To opt in to telemetry, run ``ceph
+  telemetry on``. To opt in to the leaderboard, run ``ceph config set mgr
+  mgr/telemetry/leaderboard true``. To add a leaderboard description, run
+  ``ceph config set mgr mgr/telemetry/leaderboard_description ‘Cluster
+  description’`` (entering your own cluster description).
+* core: cache-tiering is now deprecated.
```

Comment:
@zdover23 - Edited/reviewed version from 2023 June 18: core: Cache-tiering is now deprecated.

Reply:
Accepted.